eplaunch/workflows/manager.py

```diff
@@ -0,0 +1,44 @@
+# import importlib
+from importlib import util as import_util
+import inspect
+import os
+import re
+
+
+def get_workflows(workflow_path=None):
```

Comment:
So if a workflow path isn't given, we're going to just load up the "built-in" ones.  I am thinking right now that this argument should be the path to a directory that has a "workflows" subdirectory in it. (Though I'm not sure it matters at the moment.) In this way, we can give it a path to an E+ install, which should have a workflows subdirectory in it soon, and it will process the workflows there.